tox.ini

```diff
@@ -21,6 +22,8 @@ deps =
     django21:  django>=2.1,<2.2
     django22:  django>=2.2,<2.3
     django30:  django>=3.0,<3.1
+    django31:  django>=3.1,<4.0
+    django40:  django>=4.0,<5.0
```

Comment:
Is it intentional that Django 3.2 is missing here?

Reply:
I assumed that 3.2 will be covered by 3.1,<4.0 range,
Should I split this range into 2 ranges (>=3.1,<3.2 and >=3.2,<4.0)?

Reply:
I see. `django>=3.1,<4.0` will give you 3.2 then and 3.1 will not be covered. If you want both to be covered, splitting sounds like a good idea.

Reply:
PS: Based on the image below, maybe Django 2.2 + 3.2 + 4.0 is all versions worth covering in 2021-12. 

![Django releases](https://static.djangoproject.com/img/release-roadmap.3c7ece4f31b3.png)

Reply:
so no need to split the range, right?

Reply:
It should probably be `django32:  django>=3.2,<3.3` though, then. What do you think?

Reply:
Replaced django31 with django32